Gemfile

```diff
@@ -8,7 +8,7 @@ end
 gem 'spree', github: 'spree/spree', branch: 'main'
 gem 'rails-controller-testing'
 gem 'spree_auth_devise', github: 'spree/spree_auth_devise', branch: 'main'
-gem 'spree_reviews', github: 'spree/spree_reviews'
+gem 'spree_reviews', github: 'bookmebus/spree_reviews', branch: 'update-spree-450'
```

Comment:
let's wait for spree_reviews to be merged and tagged. then we point this to the tag version of 4.5 instead. The branch might ultimately deleted.